server/core/src/main/java/org/infinispan/server/core/ServerManagement.java

```diff
@@ -13,6 +13,5 @@
 
    void stop();
 
-   Set<String> cacheManagersNames();
-
+   Set<String> cacheManagerNames();
```

Comment:
@gustavonalle I think this would be the correct plural. But still not sure.

Reply:
It looks correct to me